test/proto2pb/test_all_types.pb.go

```diff
@@ -1,6 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // source: test/proto2/test_all_types.proto
 
+// Package proto2pb encapsulates the proto2 message, enum, and fields supported by CEL.
```

Comment:
This looks like a generated file - could we perhaps not check it in?

Reply:
I've checked this in mostly so that more of the CEL packages work with `go get` and so that the IDEs can easily run tests without having to invoke `bazel` build targets. Maybe not the greatest to check this artifact in, but that's also how we do things with the go genproto files for the `syntax.proto`, etc.